operations/dd-toc-guide.md

```diff
@@ -49,6 +55,57 @@ We expect all TOC members to be mindful of their obligations and timelines, whet
 
 The issue will contain a limited set of information about the project at the time of its application, commonly asserting its conformance to the stated criteria with links to where or descriptions as to how they are implemented.
 
+### Initial triage/evaluation prior to assignment
+
+All TOC members are expected to assist in the triaging of project applications to move levels to ensure that when a TOC member is ready to be assigned, the project is ready to be evaluated with no immediate blockers that would inhibit or delay the TOC's engagement.
+
+This light-weight triage/evaluation must cover the list below (it is not exhaustive and is a minimum triage set from the [incubation template retrieved 12 DEC 2025](https://github.com/cncf/toc/blob/c2943ffc98064dd88e9ef9c4afd5a8856898942f/.github/ISSUE_TEMPLATE/template-incubation-application.md)):
```

Comment:
nit: probably want to use relatively link

Reply:
I didnt use a relative link because the list is based on the content of the file from that commit. If the file changes, this list will need to also change.